src/app/control.rs

```diff
@@ -442,7 +443,7 @@ pub mod client {
                 .clone()
                 .with_remote(target.connect.addr)
                 .executor();
-            Ok(http::h2::Connect::new(c, e))
+            Ok(http::h2::Connect::new(c, e, H2Config::default()))
```

Comment:
TIOLI: might be worthwhile making a separate `h2::Connect::new` and `h2::Connect::new_with_config` or something so this doesn't have to change? I dunno.

Reply:
I don't know if I like adding a new method on `Connect` only to reduce the diff here. I prefer this way, but I can definitely make the change if I'm in the minority here.